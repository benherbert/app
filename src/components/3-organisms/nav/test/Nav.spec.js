import React from 'react'
import renderer from 'react-test-renderer'
import { Link, StaticRouter } from 'react-router-dom'

import { Nav } from '../Nav'

const context = {
  test: 'test'
}

it('renders correctly', () => {
  const component = renderer.create(
    <StaticRouter location='someLocation' context={context}>
      <Nav />
    </StaticRouter>
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
