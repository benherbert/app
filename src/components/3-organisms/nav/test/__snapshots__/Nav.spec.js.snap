// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav>
  <ul>
    <li>
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li>
      <a
        href="/about/"
        onClick={[Function]}
      >
        About
      </a>
    </li>
    <li>
      <a
        href="/users/"
        onClick={[Function]}
      >
        Users
      </a>
    </li>
  </ul>
</nav>
`;
